lib/tasks/stats.rake

```diff
@@ -574,9 +574,21 @@ namespace :stats do
         exposure: active_exp.where('TIMESTAMPDIFF(YEAR, date_of_birth, created_at) >= 70').where('TIMESTAMPDIFF(YEAR, date_of_birth, created_at) < 80').count,
         isolation: active_iso.where('TIMESTAMPDIFF(YEAR, date_of_birth, created_at) >= 70').where('TIMESTAMPDIFF(YEAR, date_of_birth, created_at) < 80').count
       }
-      results[title]['Age - >= 80'] = {
-        exposure: active_exp.where('TIMESTAMPDIFF(YEAR, date_of_birth, created_at) >= 80').where('TIMESTAMPDIFF(YEAR, date_of_birth, created_at) < 110').count,
-        isolation: active_iso.where('TIMESTAMPDIFF(YEAR, date_of_birth, created_at) >= 80').where('TIMESTAMPDIFF(YEAR, date_of_birth, created_at) < 110').count
+      results[title]['Age - 80-89'] = {
```

Comment:
Please revert this change; this is not related to the analytics page.

Reply:
done: https://github.com/SaraAlert/SaraAlert/pull/993/commits/271aea9c7f281c9b2245f3ee69e5ea924f9ecdf2